FusionIIIT/applications/finance_accounts/models.py

```diff
@@ -31,11 +31,12 @@ class Paymentscheme(models.Model):
     registrar_director_verify = models.BooleanField(default=False)
     runpayroll = models.BooleanField(default=False)
     view = models.BooleanField(default=True)
+    userId = models.CharField(max_length=20, null = True)
 

```

Comment:
you should use the get_user_model whenever link a user to a table using the Foreign Key.

Reply:
@its-me-Navneet

Reply:
@PranshuNayak  Now we dont need userid we are doing it through pf number which is unique for every employee and thus we have made the changes.

Reply:
If you have not made any changes in the models.py file remove it.